pkg/tfvars/baremetal/baremetal.go

```diff
@@ -156,6 +161,13 @@ func TFVars(libvirtURI, bootstrapProvisioningIP, bootstrapOSImage, externalBridg
 			})
 	}
 
+	var masterIgn igntypes.Config
+	if err := json.Unmarshal([]byte(ignition), &masterIgn); err != nil {
+		return nil, err
+	}
+	ignitionURL := masterIgn.Ignition.Config.Merge[0].Source
```

Comment:
So we're extracting the merge out of the stub ignition: what happens to user customizations? Does the ignition from MCS contain them?

Reply:
Yes, this is just a minor rework of extracting the URL and CA cert. Regarding user customizations, the current idea is to compare the original stub with any user customizations and, if they are different, the installer will create the appropriate machineconfig manifest for the change. This way when MCO becomes active, the final rendered ignition will contain the user's modification. This PR will go to work as usual and pass the URL and cert to the TF provider, which will extract the rendered ignition.

Reply:
Forgot to mention that there will be another PR that will  be responsible for creating the machineconfig to reconcile any changes to the stub ignition.

Reply:
Any customizations should be accessible via the MCS rendered config if we rebase this on https://github.com/openshift/installer/pull/4413